main/src/cgeo/geocaching/maps/PositionHistory.java

```diff
@@ -33,32 +32,22 @@ public PositionHistory() {
 
     // save current coords to permanent storage
     private void saveToStorage(final Location coordinates) {
-        AndroidRxUtils.andThenOnUi(Schedulers.io(), new Runnable() {
+        AsyncTask.execute(new Runnable() {
```

Comment:
According to the RxJava documentation
```suggestion
        Schedulers.io().scheduleDirect(new Runnable() {
```
seems to more appropriate, because it still uses `Schedulers.io()`, which is dedicated for I/O-bound operations.

Reply:
Today's commit changes both occurrences to `Schedulers.io().scheduleDirect`